src/index.html

```diff
@@ -3,12 +3,29 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.0/css/all.min.css">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#"><img src="./images/logo.png" alt="/"></a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="is-active nav__link" href="#">apple</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">samsung</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">smartphones</a></li>
+          <li class="nav__item"><a class="nav__link" data-qa="hover" href="#">laptops&computers</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">gadgets</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">tablet</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">photo</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">video</a></li>
```

Comment:
Code is hradly readable.
Add more lines.
One tag - separate line.
Three or more attributes - each on the separate line.

Reply:
ok, got it. fixed